default.nix

```diff
@@ -0,0 +1,12 @@
+let pkgs = import <nixpkgs> {};
```

Comment:
By including this, `nix-build` will build the `slippc` binary and output it under a `result/` symlink directory.

If someone wants to add this to their package set, they can `callPackage (import slippcSrc) {};`)

(Laziness + default arguments are fun).